invenio_rdm_migrator/streams/runner.py

```diff
@@ -31,6 +32,13 @@ def __init__(self, stream_definitions, config_filepath):
         self.tmp_dir.mkdir(parents=True, exist_ok=True)
         self.cache_dir = Path(config.get("cache_dir"))
         self.cache_dir.mkdir(parents=True, exist_ok=True)
+        self.errorlog_dir = Path(config.get("errorlog_dir"))
+        self.errorlog_dir.mkdir(parents=True, exist_ok=True)
+        logger = logging.getLogger("migration")
+        logger.setLevel(logging.ERROR)
```

Comment:
curiosity: do we need to set the level in the logger AND the file handler? wouldn't it be enough with one of them?

Reply:
yes as there are 2 levels of configuration, one for the handler and one for the logger...